pages/utils/formatApiPortfolioData.ts

```diff
@@ -0,0 +1,22 @@
+interface IFormatApiPortfolioData extends Array<object>{};
+
+export const formatApiPortfolioData = ( data: IFormatApiPortfolioData) => {
+  const result:Array<object> = [];
+
+  data.reduce((accum: any, product: any) => {
```

Comment:
```suggestion
  data.reduce((accum: any, product: any) => {
```

I cannot understand whats going on in this function

1. we should place all the routes into variables(_embedded etc.)
2. we need to split code into multiple parts to make it readable

Reply:
3 missing types